source/user-guide/getting-aws-credentials.html.md.erb

```diff
@@ -13,14 +13,14 @@ You can obtain temporary AWS credentials to use the AWS CLI or other command lin
 
 To have access to your AWS credentials you will need to be a member of the GitHub team specified when the [environment was created](./creating-environments.html#github-team-slug)
 
-Note - you do not need to obtain AWS credentials to deploy infrastructure, this is done via GitHub Workflows (see [deploying you infrastructure](deploying-your-infrastructure.html).
+Note - you do not need to obtain AWS credentials to deploy infrastructure, this is done via GitHub Workflows (see [deploying you infrastructure](deploying-your-infrastructure.html)).
```

Comment:
`[deploying your infrastructure]` instead of `[deploying you infrastructure]`?

Reply:
Good spot! Sorted 😁